lib/Interpreter/ValueExtractionSynthesizer.cpp

```diff
@@ -12,8 +12,8 @@
 #include "cling/Interpreter/Interpreter.h"
 #include "cling/Interpreter/Transaction.h"
 #include "cling/Interpreter/Value.h"
-
 #include "cling/Utils/AST.h"
+#include "cling/Utils/Output.h"
```

Comment:
What do we need this header for? (I might have missed it, or it might have been from an earlier revision of your code that didn't use the CustomDiag mechanism?)